core/src/main/java/org/ldaptive/auth/ext/PasswordPolicyAccountState.java

```diff
@@ -42,6 +53,32 @@ public PasswordPolicyAccountState(final PasswordPolicyControl.Error error)
   }
 
 
+  /**
+   * Creates a new password policy account state.
+   *
+   * @param  exp  account expiration
+   * @param  error  containing password policy error details
+   */
+  public PasswordPolicyAccountState(final ZonedDateTime exp, final PasswordPolicyControl.Error error)
+  {
+    super(new AccountState.DefaultWarning(exp, -1), error);
+    ppError = error;
+  }
+
+
+  /**
+   * Creates a new password policy account state.
```

Comment:
It might be helpful to add some text about when each constructor form should be used. I can infer that this one is for _both_ an error and warning condition, but some text may be helpful.

Reply:
Fixed.